package.json

```diff
@@ -1,12 +1,26 @@
 {
   "name": "egghead-instructor-center",
+  "scripts": {
+    "start": "npm-run-all -p start:**",
+    "start:api": "json-server -w .fakeApi/index.js -r .fakeApi/routes.json -p 4000",
```

Comment:
I plan to default to this in the future but allow passing an environment option to `yarn start` so that `start:api` can run differently depending on the environment:
- Nothing, default to `fake`
- `fake` => Run this `json-server` stuff
- `dev` => Run `egghead-sytems` vagrant up + vagrant ssh + cd /app + foreman start (and add note to README that `egghead-instructor-center` should be cloned as a sibling to `egghead-systems` and `egghead-rails`
- `prod` => don't run anything